test/service/test_manifest.py

```diff
@@ -431,10 +431,13 @@ def test_manifest_zarr(self):
                 'specimen_from_organism_organ': 'brain'
             }
         ]
-        for entry in expected:
-            url = furl(entry['file_drs_uri'])
-            entry['file_drs_uri'] = url.set(scheme='drs',
-                                            host=config.drs_domain).url
+
+        def _file_drs_uri(drs_uri):
+            drs_uri = furl(drs_uri)
+            self.assertEqual(drs_uri.scheme, 'drs', drs_uri.origin)
+            self.assertIsNotNone(drs_uri.host)
```

Comment:
This would pass if the host name was the empty string which is not a valid host name. Is that your intention?

Reply:
> Is that your intention?
No, I wanted to assert that it is something which isn't None (or empty string).